PassleSync.Website/App_Plugins/PassleSync/backoffice/passleSync/components/organisms/entitiesTable/entitiesTable.directive.js

```diff
@@ -95,26 +95,33 @@
                 }
 
                 syncProm.then((response) => {
-                    this.entities.forEach((entity, ii) => {
-                        let matchingEntities = response[this.entityReponseParam].filter(x => x.Shortcode === entity.shortcode);
-                        if (matchingEntities.length > 0) {
-                            this.entities[ii] = getRowData(matchingEntities[0]);
-                        }
-                    });
+                    if (response[this.entityReponseParam].length > 0) {

```

Comment:
Can we invert this if and return early so we're not wrapping everything in an extra layer of indentation? e.g.
```js
if (!response[this.entityReponseParam].length) {
  this.isUpdating = false;
  notificationsService.error("Error", $scope.entityInfo.namePlural + " failed to sync");
  return;
}
```